src/Components/SingleBeer/SingleBeer.js

```diff
@@ -1,40 +1,41 @@
 import React, { useEffect, useState } from 'react';
-import { useParams } from 'react-router-dom'
+import { useParams, useLocation } from 'react-router-dom'
 import './SingleBeer.scss'
 
-const SingleBeer = ({ beerData }) => {
-  console.log(beerData)
-  const [singleData, setSingleData] = useState('');
-  let currentId = useParams().beerId
-
-  const getSingleBeer = () => {
-    const currentBeer = beerData.find((beer) => {
-      return beer.id === parseInt(currentId)
-    })
-    return currentBeer
-  }
-
-  const fetchSingleBeer = async (url) => {
-    try {
-      const res = await fetch(url)
-      const resJson = await res.json()
-      setSingleData(resJson)
-      console.log(singleData)
-    } catch (error) {
-      console.log(error)
-    }
-  }
+const SingleBeer = () => {
+  const [singleData, setSingleData] = useState([]);
+  let beerId = useLocation().pathname
+  // console.log(beerId)
+
+  // const getSingleBeer = () => {
+  //   const currentBeer = beerData.find((beer) => {
+  //     return beer.id === parseInt(beerId)
+  //   })
+  //   return currentBeer
+  // }
```

Comment:
I understand why you'd want to keep these lines around for a while after refactoring, in case the new method doesn't work, but that's what your commit history is for! Go ahead and scrap 'em, says I.